lib/jsdom/level1/core.js

```diff
@@ -12,7 +12,7 @@ var HtmlToDom = require("../browser/htmltodom").HtmlToDom;
 var Location = require("../browser/location");
 var vm = require("vm");
 var CookieJar = require('tough-cookie').CookieJar;
-var EventTarget = require("../events/EventTarget");
+var EventTarget = require("../web-idl/generated/EventTarget");
```

Comment:
I'd prefer ../living/generator. web-idl folder is for things defined in the actual Web IDL spec, like DOMException.